TATB - 37 : Redesign Quiz Test Component

PrabhatJTDeveloper opened the pull request.

PrabhatJTDeveloper committed:
- TATB-37 redesign the quiz test template
  - for that change the html and scss file of the quiz component .

PrabhatJTDeveloper committed:
- TATB 37
   - removed unnecessary id and role in the button from the html file
   - removed unwanted css in scss file

PrabhatJTDeveloper committed:
- TATB 37
   - removed unwanted id and class from carousel in html file
   - removed the css which is also used as boostrap class.

PrabhatJTDeveloper committed:
- TATB 37
  - conflicts occurs in quiz.component html and scss file so solve it
  - change card.value with card.id in quiz component html file

PrabhatJTDeveloper committed:
- TATB 37 removed unused div in html file

PrabhatJTDeveloper committed:
- TATB 37
  - make change in condition of hightlighted text in quiz.component.html file

PrabhatJTDeveloper committed:
- TATB 37
   - Solved the conflicts in quiz component html and css file.

charmijtdev commented on src/app/component/quiz/quiz.component.scss:
```diff
@@ -1,57 +1,73 @@
-  border-radius: 5px;
+
+.que_txt {
+  padding: 15px 15px;
```
can we use p-3 ?

PrabhatJTDeveloper replied:
yes, i changed it

charmijtdev commented on src/app/component/quiz/quiz.component.scss:
```diff
@@ -1,57 +1,73 @@
+.option {
+  background: rgb(222, 239, 255);
+  border: 1px solid #84c5fe;
+  padding: 8px 15px;
```
same as can we manage with p class ?

PrabhatJTDeveloper replied:
yes, i changed it.

charmijtdev commented on src/app/component/quiz/quiz.component.scss:
```diff
@@ -1,57 +1,73 @@
+  padding: 8px 15px;
+  font-size: 17px;
+  cursor: pointer;
+  margin-top: 15px;
```
please use mt-4

PrabhatJTDeveloper replied:
yes, i changed it.

PrabhatJTDeveloper committed:
- TATB 37
   - give the boostrap class and remove from the css
   - and make change in html file

PrabhatJTDeveloper committed:
- TATB 37
   - make changes in html and scss file.

charmijtdev commented on src/app/component/quiz/quiz.component.html:
```diff
@@ -1,108 +1,145 @@
+                      class="card-header header_txt py-0 px-3 d-flex align-items-center justify-content-between position-relative"
+                    >
+                      <div class="header_title user-select-none">
+                        {{ [this.selectedQuizType] }}
```
why used like this and never use this inside of html file

PrabhatJTDeveloper replied:
yes, i changed it.

PrabhatJTDeveloper committed:
- TATB 37
   - removed "this" from the {{this.selectedQuizType}} in the quiz component html file.

PrabhatJTDeveloper committed:
- TATB 37
  - removed "this" from the html file
